pom.xml

```diff
@@ -4,37 +4,27 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>3.48</version>
+        <version>4.19</version>
         <relativePath />
     </parent>
 
     <artifactId>junit-attachments</artifactId>
     <version>${revision}${changelist}</version>
     <packaging>hpi</packaging>
     <name>JUnit Attachments Plugin</name>
-    <url>https://wiki.jenkins.io/display/JENKINS/JUnit+Attachments+Plugin</url>
-    <issueManagement>
-        <system>JIRA</system>
-        <url>https://issues.jenkins-ci.org/secure/IssueNavigator.jspa?reset=true&amp;jqlQuery=project+%3D+JENKINS+AND+status+in+%28Open%2C+"In+Progress"%2C+Reopened%29+AND+component+%3D+${project.artifactId}</url>
-    </issueManagement>
+    <url>https://github.com/jenkinsci/junit-attachments-plugin</url>
 
-	<scm>
-		<connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
-		<developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
-          <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
-	  <tag>${scmTag}</tag>
-  </scm>
+    <scm>
+        <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
+        <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
+        <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
```

Comment:
BTW the new `gitHubRepo` property is recommended in case you are incrementalified (affects use of PCT on forked PRs). Likely irrelevant here.